src/Containers/SlidingOutPanels.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MainInterface, ResponsiveInterface } from '../Utils/BaseStyles';
+
+export interface SlidingOutPanelProps
+    extends MainInterface,
+        ResponsiveInterface {
+    imageLink: string;
```

Comment:
What was the design decision behind this? Why not an Array of Images with Key, ImageSource, ImageLink for full customizability.

Reply:
Initially I was using a single image across for all the different animations but for customization an array would be more suitable, I shall make this change.